conans/client/grapher.py

```diff
@@ -0,0 +1,39 @@
+import os
+from conans.model.ref import ConanFileReference
+
+class ConanGrapher(object):
+    def __init__(self, project_reference, deps_graph):
+        self._deps_graph = deps_graph
+        self._project_reference = project_reference
+
+    def graph(self):
```

Comment:
Not necessarily! :) I was just saying about a small internal detail. ``--graph`` printing to stdout has other problems, like extra command line syntax, messing with other stdout, etc. I like it output to file (as a command). I was just saying about the internal implementation of the grapher, one method returns text, other method calls it and save to file, with a filename given as parameter (in order to avoid "magic strings"). Such parameter can be hard-coded in ``command.py`` by now, not big issue, we will change it later if users request so.

Thanks again for your effort, and don't worry if you are busy, I can just contribute a PR to yours with those minimum changes, you already did the hard work.